core/src/database/ProxyBackend.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ * ProxyBackend.h
```

Comment:
Shouldn’t this be a `.hpp`?

Reply:
https://github.com/pollastri-pierre/lib-ledger-core/commit/601dd74a4c566ba3aaebac2e488a4a30fdd8dadc